solutions/community-deploy/cloud/AWS/variable.tf

```diff
@@ -41,7 +41,7 @@ variable "key_name" {
 variable "instance_count" {
   description = "EC2 instance count"
   type        = number
```

Comment:
remove extra spaces from `type` and `=` sign

Reply:
fixed